events/scheme_events/stasis_trap_scheme/wc_stasis_trap_outcome_events.txt

```diff
@@ -0,0 +1,351 @@
+		if = {
+			limit = {
+				exists = scope:scheme_discovered
+				scope:scheme_discovered = yes 
+			}
+
+			stasis_trap_success_effect = { DISCOVERED = scope:scheme_discovered }
+		}
+		else = {
+			stasis_trap_success_effect = { DISCOVERED = no }
+		}
+		
+	}
+
+	option = {
+		name = stasis_trap_outcome.2001.a
+		play_music_cue = "mx_cue_prison"
+		successful_stasis_trap_outcome_event_option_effect = yes
+	}
+}
+
+#################################
+# FAILURE OUTCOMES 4000 - 4999
+#################################
+stasis_trap_outcome.4001 = {
+	type = character_event
+	title = stasis_trap_outcome.1001.t
+	desc = {
+		desc = stasis_trap_outcome.4001.opening
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					exists = scope:scheme_discovered
+				}
+				desc = stasis_trap_outcome.4001.owner_discovered
+			}
+			desc = stasis_trap_outcome.4001.owner_secret
+		}
+	}
+	theme = intrigue
+	left_portrait = {
+		character = scope:target
+		animation = rage
+	}
+	widget = {
+		gui = "event_window_widget_scheme"
+		container = "custom_widgets_container"
+	}
+	
+	trigger = {
+		scope:stasis_trap_method = flag:trapped_residence
+	}
+
+	immediate = {
+		stasis_trap_failure_effect = yes
+	}
+
+	option = { # Abandon scheme
+		name = stasis_trap_outcome.4001.a
+		abandon_scheme_effect = yes
+
+		ai_chance = {
+			base = 100
+		}
+	}
+
+	option = { # Try again!
+		name = stasis_trap_outcome.4001.b
+		restart_scheme_effect = yes
+
+		ai_chance = {
+			base = 10
+			modifier = {
+				add = 200
+				has_trait = stubborn
+			}
+		}
+	}
+}
+
+########################################
+# TARGET SUCCESS NOTIFICATIONS
+########################################
+
+#I have been stunned!
+stasis_trap_outcome.5001 = {
+	type = character_event
+	title = stasis_trap_outcome.5001.t
+	desc = {
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					scope:scheme_discovered = yes
+				}
+				desc = stasis_trap_outcome.5001.opening.b
+			}
+			triggered_desc = {
+				trigger = {
+					scope:scheme_discovered = no
+				}
+				desc = stasis_trap_outcome.5001.opening.a
+			}
+		}
+		first_valid = {
+			triggered_desc = {
+				trigger = {
+					scope:stasis_trap_method = flag:trapped_residence
+				}
+				desc = stasis_trap_outcome.5001.trapped_residence
+			}
+		}
+	}
+	theme = prison
+	left_portrait = {
+		character = scope:owner
+		animation = scheme
+	}
+
+	immediate = {
+		show_as_tooltip = {
+			stun_character_effect = {
+				TARGET = scope:target
+				STUNNER = scope:owner
+			}
+		}
+	}
+
+	option = {
+		name = stasis_trap_outcome.5001.a
+		trigger = {
+			scope:scheme_discovered = yes
+		}
+		play_music_cue = "mx_cue_stress"
+		hostile_scheme_opinion_effect = {
+			VICTIM = scope:target
+			ATTACKER = scope:owner
+			SCHEME = scope:scheme
+		}
+	}
+
+	option = {
+		name = stasis_trap_outcome.5001.a
+		trigger = {
+			OR = {
+				scope:scheme_discovered = no
+				NOT = { exists = scope:scheme_discovered }
+			}
+		}
+		play_music_cue = "mx_cue_stress"
+	}
```

Comment:
Both options are visible. I guess it's related to using `yes` or `no` I reported above.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117648544-66f23d80-b19f-11eb-94c0-5444adf16356.png)

</details>